### ISSUE TITLE
Difficulty with malloc error

### ISSUE THREAD

**Issue by rahulcdeo** [yarden/MISO#46]
Hi,

I'm running Miso on Mac OS X 10.8.3 and am having frequent errors of this sort:

Python(57341) malloc: **\* error for object 0x104120d00: incorrect checksum for freed object - object was probably modified after being freed.

I am running the --compute-genes-from-file step and looking at alternative splicing events.

Thanks

**Comment by yarden** [comment]
Hi,

What MISO version are you using? Did you try it with `misopy-0.4.9` (latest)? Also, what version of GCC? Are you running it through any pipeline that uses `make -j`?

We've had a few users report this type of error, all on Mac OS X, but we've never been able to reproduce it, so any information you might have will be helpful. In the meantime, I suggest using Linux if possible.

  --Yarden

**Comment by rahulcdeo** [comment]
Hi,

This is the latest github clone.  OS X 10.8.3 no longer supports gcc4.2 so I used gcc4.7.

I examined the logfiles and noticed that there were only a handful of events (~5/1500) that were being considered that led to the crash.  These all take place right after multiple reads are eliminated by filtering (same strand, no pairs, etc.,) - I wonder if there is some division by 0 problem that's occurring once filtering takes place.  

I'm finding far fewer (if any) problems when I enable the pre-filter option.  I'll switch over to my Linux cluster once I get Numpy and Scipy installed

Rahul

**Comment by gaborcsardi** [comment]
Hi, can you share a (preferably) small GFF and SAM/BAM file that reproduces
the error? Thanks.

**Comment by rahulcdeo** [comment]
Yes - I'll do so a little later today (am tied up in meetings).  I should also have some of the log files which captured the crash.

Best,
Rahul
